src/GeositeFramework/js/Map.js

```diff
@@ -122,16 +133,26 @@ require(['use!Geosite',
                     }
             }, 300),
             loadEventFired = false;
+        */
+        var esriMap = new Map({ basemap: 'topo' });
```

Comment:
The framework previously allowed setting the initial basemap via configuration in the region.json (uses the first in the list of basemap layers) - I don't think we _need_ to retain that in this pilot branch, but it'd be good to at least leave a comment that there's some expediency being taken here.